app/services/adapters/structs/capital_details.rb

```diff
@@ -8,7 +8,7 @@ def savings
       end
 
       def serializable_hash(options = {})
-        super options.merge(except: [:savings])
+        super options.merge(except: [:savings, :investments])
       end
```

Comment:
@timpeat this is a temporary update pending the update of Capital Details (datastore/schema/etc) I think